### PR TITLE
Added JenkinsFile

### PR DESCRIPTION
**Pull request by sladyn98**
Closes https://github.com/sladyn98/JCasC-VSCode-Plugin/issues/13

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+pipeline {
+    agent { docker { image 'node:6.3' } }
```
Comment (timja) [inline Jenkinsfile]: This is soo ancient, surely vs code doesn’t recommend this?

Comment (sladyn98) [reply Jenkinsfile]: Should we use 10 ?

Comment (timja) [reply Jenkinsfile]: Yes

Comment (sladyn98) [reply Jenkinsfile]: Changed

**Pull request merged**